### DISCUSSION TITLE
FreeBSD:  Installing Snipe-IT in FreeBSD jails

### DISCUSSION THREAD

**Discussion by gladiola** [snipe/snipe-it#9446] [Q&A]
### Are there any directions or notes on installing Snipe-IT on FreeBSD?  

I did see one file in the repo which seemed to reference FreeBSD, but I didn't find installation instructions for that directly.  I was planning on using the manual install instructions, and slowly modifying them as needed.  If anyone has any tips that could save me a little time, that would be helpful.  I would be particularly interested in any tips that defined if I need to install Linux emulation or not; I'd prefer to leave it out, if possible.

I suspect that the program will not need any Linux-specific structures; it seems like it will run on the web server with the required db and php components.  I'll make notes and maybe a shell script as I go.  I will plan to run Snipe-IT in a thick jail with a dedicated IP and some pf rules.  

Thanks in advance for any help the community might provide.  This looks like it's going to be a really good program to use.  I wish I had seen it a year ago.

Respectfully,
j.

**Comment by VersusBG** [comment]
In the Discussions i posted a short how to install on FreeBSD but it's not for a jail. You can see it here: https://github.com/snipe/snipe-it/discussions/9123

**Reply by gladiola** [reply]
Thank you for your time and attention.
